src/backend/dx12/src/resource.rs

```diff
@@ -389,6 +392,48 @@ pub struct Memory {
 unsafe impl Send for Memory {}
 unsafe impl Sync for Memory {}
 
+bitflags! {
+    /// A set of D3D12 descriptor types that need to be associated
+    /// with a single gfx-hal `DescriptorType`.
+    #[derive(Default)]
+    pub struct DescriptorContent: u8 {
+        const CBV = 0x1;
+        const SRV = 0x2;
+        const UAV = 0x4;
+        const SAMPLER = 0x8;
+        const VIEW = 0x7;
```

Comment:
`const VIEW = Self::CBV.bits | Self::SRV.bits | Self::UAV.bits;`

Reply:
unfortunately that doesn't work: it expects `u8` but the expression is of type `Self`

Reply:
should be working: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=9abc4c17f80253a0e4e2b809c81bbfac